bundles/media/imagine-adapter.rst

```diff
@@ -71,7 +77,6 @@ To install it use following commands:
 .. code-block:: bash
 
     apt-get install libvips-dev
-    pecl install vips
```

Comment:
VIPS used FFI and not longer requires a PHP extension: https://github.com/libvips/php-vips-ext